qrhl/qrhl-input.el

```diff
@@ -0,0 +1,766 @@
+;;; qrhl-input.el --- Quail package for TeX-style input for qrhl-tool in ProofGeneral -*-coding: utf-8;-*-
+
+;; Copyright (C) 2001-2018 Free Software Foundation, Inc.
+;; Copyright (C) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009,
+;;   2010, 2011
+;;   National Institute of Advanced Industrial Science and Technology (AIST)
+;;   Registration Number H14PRO021
+
```

Comment:
You might want to add a Copyright line for your changes here(?)

Reply:
I don't understand enough about the legal ramifications of GPL to figure out who owns the copyright here. (What does it even practically mean to have the copyright in GPL'd source...) So I leave it as it is. Added myself as an author since that is something I understand.